client/state/login/actions.js

```diff
@@ -137,7 +137,7 @@ export const loginUserWithTwoFactorVerificationCode = ( user_id, two_step_code,
  * @return {Function}                Action thunk to trigger the request.
  */
 export const sendSmsCode = ( userId, twoStepNonce ) => dispatch => {
-	return request.post( 'https://wordpress.com/wp-login.php?action=send-sms-code-endpoint' )
+	return request.post( config( 'two_step_authentication_send_sms_code_xhr' ) )
```

Comment:
It looks like `login_url` changes per environment, but `two_step_authentication_send_sms_code_xhr` and `login_url_xhr` do not (and we probably just followed the pattern set by `login_url` before without considering it). I'll add a commit to remove these from `config`.